src/components/Drawer/index.tsx

```diff
@@ -21,6 +21,7 @@ import { ResumeConfig, ThemeConfig } from '../types';
 import { ConfigTheme } from './ConfigTheme';
 import { Templates } from './Templates';
 import './index.less';
+import useThrottle from '@/hooks/useThrottle';
```

Comment:
https://ahooks.js.org/hooks/use-throttle

```suggestion
import { useThrottle } from 'ahooks';
```

Reply:
use `react-use` instead of `ahooks`

Reply:
LGTM